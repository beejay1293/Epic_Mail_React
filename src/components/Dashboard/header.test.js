import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Header from './header';
describe('Dashboard Header', () => {
  it('should render correctly the dashboard header', () => {
    const Head = shallow(<Header />);
  
    expect(Head).toMatchSnapshot();
  });
});
