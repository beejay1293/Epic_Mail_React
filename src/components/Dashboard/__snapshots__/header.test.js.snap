// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Header should render correctly the dashboard header 1`] = `
<header
  className="header__container"
>
  <div
    className="logo"
  >
    <h1
      className="header"
    >
      <div
        className="bars"
      >
        <h6
          className="nav_name"
        />
      </div>
      <div
        className="epic__mail"
      >
        <span>
          <i
            className="fas fa-envelope"
          />
          <em>
            E
          </em>
        </span>
        PIC
        <b
          className="mail"
        >
          Mail
        </b>
      </div>
    </h1>
  </div>
  <div
    className="navbar"
  >
    <ul
      className="dropdown"
    >
      <i
        className="far fa-user"
      >
        <h5
          className="user"
        >
          Mobolaji
          <i
            className="fas fa-caret-down"
          />
        </h5>
        <div
          className="dropdown-content"
        >
          <a
            className="reset__password"
            href="/login"
          >
            logout
          </a>
        </div>
      </i>
    </ul>
  </div>
</header>
`;
